src/components/display/Tooltip.jsx

```diff
@@ -52,25 +52,42 @@ const TooltipWrapperWithCss = styled(TooltipWrapper)`
 `;
 
 const Tooltip = React.forwardRef(function TooltipFn(
-	{ label, placement, maxWidth, children, disabled, disablePortal, overflowTooltip, ...rest },
+	{
+		label,
+		placement,
+		maxWidth,
+		children,
+		disabled,
+		disablePortal,
+		overflowTooltip,
+		triggerDelay,
+		...rest
+	},
 	ref
 ) {
 	const [open, setOpen] = useState(undefined);
 	const popperInstanceRef = useRef(undefined);
 	const triggerRef = useRef(undefined);
 	const innerRef = useRef(undefined);
 	const tooltipRef = useCombinedRefs(ref, innerRef);
+	const timeout = useRef(undefined);
```

Comment:
also, i would prefer if the variable name was timeoutRef, just to keep the consistency of the naming